centaur/src/main/resources/reference.conf

```diff
@@ -35,7 +35,7 @@ centaur {
   # The timeout of the Centaur send/receive + unmarshal pipeline
   sendReceiveTimeout: 10 seconds
   # The maximal length of a workflow, intended as a sanity check and not actually a test in and of itself
-  maxWorkflowLength: 3 hours
+  maxWorkflowLength: 4 hours
```

Comment:
This doesn't change anything afaik and may give folks a false sense of security. Travis is still going to 🔫  your jobs after 3 hours. Maybe if/when we move to another CI we'll get more time though.

Reply:
true-- will revert!